runner/config.go

```diff
@@ -83,6 +78,30 @@ func initConfig(path string) (cfg *config, err error) {
 	return cfg, err
 }
 
+func defaultPathConfig() (*config, error) {
+	// when path is blank, first find `.air.toml`, `.air.conf` in `air_wd` and current working directory, if not found, use defaults
+	for _, name := range []string{dftTOML, dftConf} {
+		var path string
+		if wd := os.Getenv(airWd); wd != "" {
```

Comment:
if user use dftConf. can we print a warning log to tell user, .conf will be Deprecated soon, recommend to use .toml .

Reply:
@xiantang Added!
https://github.com/cosmtrek/air/pull/90/commits/9216bb2d99ac3603ec93be9d6944ab6e7feb0055